modules/functions.js

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable max-len */
+let toDoList = [];
+
+// function to add the set tasks to local storage
+const addToLocalStorage = (task) => {
+  toDoList = JSON.parse(localStorage.getItem('tasks: '));
+  if (toDoList == null) { toDoList = []; }
+  toDoList.push(task);
+  localStorage.setItem('tasks: ', JSON.stringify(toDoList));
+};
+
+// const inputVal = document.getElementById('add');
```

Comment:
- [OPTIONAL] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.